packages/main/src/plugin/color-registry.spec.ts

```diff
@@ -433,7 +434,7 @@ describe('registerExtensionThemes', () => {
     expect(colors).toBeDefined();
     titlebarBg = colors.find(c => c.id === 'titlebar-bg');
     expect(titlebarBg).toBeDefined();
-    expect(titlebarBg?.value).toBe('#0f0f11');
+    expect(titlebarBg?.value).toBe(isWindows() ? '#202020' : '#0f0f11');
```

Comment:
here I think the unit test should mock the windows condition

like we should have 2 tests,
one checking the expected value when it'll return Windows and one for the other

Else it means that if I'm on mac/Linux, the test will pass locally but only fails on the CI environment, while I would expect that I'm able to check the non-regression even if I'm on Linux/macOS

Reply:
It looks like it has passed, but I can split it to Windows and others test
Something like: 
`if (isWindows()) {test(windowsTest)} 
else {test(otherTest)}`

Reply:
it should be something like:

```
test('check on Windows', () => {
mock util to return isWindows to true
check value
})

test('check on macOs/Linux', () => {
mock util to return isWindows to false
check value
})
```